wayland-backend/src/sys/mod.rs

```diff
@@ -90,6 +95,22 @@ impl client::Backend {
     pub fn display_ptr(&self) -> *mut wayland_sys::client::wl_display {
         self.backend.display_ptr()
     }
+
+    /// Take over handling for a proxy created by a third party.
+    ///
+    /// # Safety
+    ///
+    /// There must never be more than one party managing an object. This is only
+    /// safe to call when a third party is transferring ownership of the proxy.
```

Comment:
Also, this should have a note regarding the fact that the interface is not checked, and while giving a wrong interface should not cause memory unsafety, it will definitely cause protocol errors.

Reply:
Yeah, I'm not sure what would be the best wording for that. What I mean is a proxy which has not yet been assigned a "listener" (using libwayland vocabulary).

Reply:
Unmanaged might be better than uninitialized?